app/Views/Templates/components/forms/text-input.blade.php

```diff
@@ -0,0 +1,48 @@
+@props([
+    'inputType' => 'text',
+    'id' => '',
+    'name' => '',
+    'placeholder' => '',
+    'label' => '',
```

Comment:
Lets call label labelText to avoid any collision with html attribute names. See our spreadsheet for details.